stable/admin/values.yaml

```diff
@@ -109,7 +109,16 @@ admin:
   ## Admin options
   # These are applied using the nuoadmin startup command
   # Add these values as appropriate for this domain
-  options: {}
+  options:
+      # this high reconnect timeout is necessary because an admin can be
+      # rescheduled on a different Node and a reconnecting engine will not
+      # connect to it until the stale DNS entry in its local cache expires,
+      # which has a default TTL of 30 seconds
+      pendingReconnectTimeout: 60000
+      # this is necessary because killed engine Pods may not generate a TCP_RST
+      # packet to close the admin connection, leaving a database process entry
+      # in the domain state that has no running Pod associated with it
+      processLivenessCheckSec: 15
```

Comment:
Now that we are not relying on this to make the tests pass, this should be bumped to something higher like 30 seconds.